api/v1/helpers/verbs/findUtils.js

```diff
@@ -196,6 +198,12 @@ function toSequelizeWhere(filter, props) {
       else if (props.tagFilterName && key === props.tagFilterName) {
         const tagArr = filter[key];
         Object.assign(where, toWhereClause(tagArr, props));
+      } else if (props.timePeriodFilters &&
+        props.timePeriodFilters.includes(key)) {
+        const whereClause = {};
+        whereClause[Op.lte] = new Date();
```

Comment:
is this "earlier than now" part of the where clause necessary, since every date will be earlier than now?

Reply:
Yeah, this was required for testing, because I am mocking the now time to be something in past  and some things like admin user is created at real now. I added this later on as a part of my iterative workarounds to get the tests working.